website/content/contributors/getting-started.md

```diff
@@ -0,0 +1,130 @@
+
+### Understand the Project
+
+Using an open source project before becoming a contributor is essential to understanding the project's functionality and architecture. If available, follow the project's tutorial or getting started documentation. Then, try it out yourself to become familiar with how the project works and its capabilities. It helps contributors identify the project's issues and challenges, allowing them to get acquainted with the codebase and its design principles. By testing the project, contributors can provide valuable feedback to the development team, which can help improve the project's quality and usability. Additionally, testing the project can help contributors identify areas where they can contribute to the project, allowing them to make more significant contributions and become more effective contributors.
+
+### Read the Documentation
+
+Before you start contributing to a project, it's important to read the documentation. Most projects have a README file that provides an overview of the project, how to set it up, and how to contribute. Read the documentation carefully to make sure you understand the project's goals, coding standards, and contribution guidelines.
+
+It's also important to invest time in reading the project's mailing list threads, community forum discussions, and meeting minutes, as well as reviewing open and closed issues and pull requests in the project repository. By doing so, you can better understand the project's context, goals, and challenges, and develop a sense of the community's culture and values. This knowledge can help you ask more informed and relevant questions when reaching out to the community, and increase your chances of getting a helpful and constructive response.
+
+### A language of their own
+
+The open source community can sometimes be overwhelming with its acronyms and technical terms. Newcomers might find it difficult to keep up with the jargon. However, there are ways to overcome this, such as taking the time to research and ask questions. Also, the Cloud Native Computing Foundation (CNCF) provides a helpful [glossary of commonly used terms](https://glossary.cncf.io/abstraction/) in the cloud native landscape. By utilizing this resource and proactively in seeking information, newcomers can better understand the community and its practices. Open-source communities aim to be collaborative and inclusive, and their members are generally welcoming to those willing to learn and contribute.
```

Comment:
Fixing link to glossary so that it goes to the homepage and not a specific term.

```suggestion
The open source community can sometimes be overwhelming with its acronyms and technical terms. Newcomers might find it difficult to keep up with the jargon. However, there are ways to overcome this, such as taking the time to research and ask questions. Also, the Cloud Native Computing Foundation (CNCF) provides a helpful [glossary of commonly used terms](https://glossary.cncf.io/) in the cloud native landscape. By utilizing this resource and proactively in seeking information, newcomers can better understand the community and its practices. Open-source communities aim to be collaborative and inclusive, and their members are generally welcoming to those willing to learn and contribute.
```